pages/api/rustApp/setup.ts

```diff
@@ -0,0 +1,20 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { saveTopicName } from '../../../utils/db/relevance';
+
+const setupHandler = async (req: NextApiRequest, res: NextApiResponse) => {
+    console.info("Saving setup info in db...");
+    const jsonbody = req.body;
+    for (const idx in jsonbody.info) {
+        let owner_info = jsonbody.info[idx];
+        try {
+            await saveTopicName(owner_info.owner, owner_info.provider, 
+                jsonbody.installation_id, owner_info.repos);
```

Comment:
I think you still missed this file in your refactor. I still see snake case (`owner_info`) and lowercase runon (`jsonbody`) in this file 🤦🏼